bundles/net.tourbook.tests/src/test/device/hac4linux/tour/HAC4LinuxDeviceReaderTests.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (C) 2022 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package device.hac4linux.tour;
+
+import net.tourbook.device.hac4linux.tour.HAC4LinuxDeviceReader;
+
+import org.junit.jupiter.api.Test;
+
+import utils.DeviceDataReaderTester;
+import utils.FilesUtils;
+
+public class HAC4LinuxDeviceReaderTests extends DeviceDataReaderTester {
+
+   public static final String    FILES_PATH       = FilesUtils.rootPath + "device/hac4linux/tour/files/"; //$NON-NLS-1$
+
+   private HAC4LinuxDeviceReader deviceDataReader = new HAC4LinuxDeviceReader();
+
+   /**
+    * Sample provided by Wolfgang Schramm
+    * https://github.com/mytourbook/mytourbook/discussions/939#discussioncomment-3730768
+    */
```

Comment:
This is probably copy and paste ;-)
The link to comment, where the file was provided, is this one:
https://github.com/mytourbook/mytourbook/discussions/939#discussioncomment-3714641

The method name of the test would be probably better changed to:
`testHac4LinuxImport_20060426`

Reply:
@cyc1ingsir 
This one is a different test. I think you're thinking about [this one ](https://github.com/FJBDev/mytourbook/blob/6b2ce726af10d913c609a0448653db89a6d2673e/bundles/net.tourbook.tests/src/test/device/hac4linux/tour/HAC4LinuxDeviceReaderTests.java#L36)no ?

Let me know if I'm missing something